fsnotify/fsevents
# ci: Re-enable CircleCI test

#### What does this pull request do?
Re-enable CircleCI test and coverage.

#### Where should the reviewer start?
Check whether the ci test succeeds or fail.

#### How should this be manually tested?
No need manually test because for CI build.

Note:
- Use `xcode 8.0`(latest) only because CircleCI does not support matrix build.
- Disable brew automatically update with `HOMEBREW_NO_AUTO_UPDATE=1`.
  Also use direct url install for fetch latest go stable version.

Close https://github.com/fsnotify/fsevents/issues/21

---

I'm not sure https://github.com/fsnotify/fsevents/issues/21 issue. Is this just want to `circle.yml` that same as `.travis.yml` test?
If so, maybe will be able to start the test. If I'm misunderstood, please ignore and close this pull request.
Note that I do not know whether the `environment` section is correct. and did not test because I can't local testing.
